packages/edit-site/src/components/dataviews/filter-enumeration.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { privateApis as componentsPrivateApis } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import { OPERATOR_IN } from './constants';
+
+const { DropdownMenuCheckboxItemV2: DropdownMenuCheckboxItem } = unlock(
+	componentsPrivateApis
+);
+
+export default function ( { filter, view, onChangeView } ) {
+	const filterInView = view.filters.find( ( f ) => f.field === filter.field );
+	const activeElement = filter.elements.find(
+		( element ) => element.value === filterInView?.value
+	);
+
+	return filter.elements.map( ( element ) => {
+		return (
+			<DropdownMenuCheckboxItem
```

Comment:
As also discussed previously (and recently [noted](https://github.com/WordPress/gutenberg/pull/56479#issuecomment-1832188914) by @andrewhayward ), mutually exclusive options should be radio menu items, instead of checkbox menu items